packages/babel-traverse/src/scope/lib/renamer.ts

```diff
@@ -26,6 +27,22 @@ const renameVisitor: Visitor<Renamer> = {
       }
     }
   },
+  ObjectProperty(path: NodePath<ObjectProperty>) {
```

Comment:
In this function we should also check if `value.name === state.oldName`, because we only want to modify the nodes that we are renaming.

Reply:
Maybe it doesn't matter? Because this should not happen normally in AST.